src/Http/Response/BadRequest.php

```diff
@@ -0,0 +1,44 @@
+<?php declare(strict_types=1);
+/**
+ * File contains the PSR15 ResponseInterface implementation for
```

Comment:
```suggestion
 * File contains the PSR7 ResponseInterface implementation for
```

ResponseInterface is part of PSR7, not 15.

Reply:
(I only requested changes on the one file, but the comments for all of the files have this problem.)